backend_api/README.md

```diff
@@ -111,4 +96,19 @@ take several minutes.
 
 Visit `https://[YOUR_PROJECT_ID].appspot.com` to view your deployed application.
 
-You can continue to make new versions of the application and deploy them with the above command.
\ No newline at end of file
+You can continue to make new versions of the application and deploy them with the above command.
+
+## Running Locally (Optionally)
+Alternatively, if you do not wish to use the Cloud Shell you can setup the Cloud SDK on your local machine instead.
+
+Download the Cloud SDK here https://cloud.google.com/sdk/docs/downloads-interactive
+
+Once downloaded and installed login using:
+
+    gcloud auth login
+
+Link to the correct project (using the wrong project might accidently bill you instead). Extra documentation https://cloud.google.com/sdk/docs/initializing
+
+    gcloud init
+
+Then follow the instructions as normal
```

Comment:
If using mac/linux:

- Follow the instructions as normal. 

If using Windows:

- Follow the instructions but replace 'export' with 'set'
- To activate your venv, just do (path)\venv\Scripts\activate